.travis.yml

```diff
@@ -8,15 +8,16 @@ python:
 
 env:
   - TOXENV=django18
-  - TOXENV=django19
-  - TOXENV=django110
+  - TOXENV=django111
 
 matrix:
   include:
     - python: 3.5
       env: TOXENV=quality
     - python: 3.5
       env: TOXENV=docs
+    - python: 3.5
+      env: TOXENV=django20
 
```

Comment:
Tangential change: edx is jumping from django 1.8 to django 1.11, so there's no reason to test against 1.9 and 1.10.  Django 2.0 came out so test it too, but it only works with python 3, not python2.

Reply:
@ciuin - Here's where I'm changing the tox environments to run in.  Feel free to use the same config on your branch, to keep merges easier.